script/makefiles/deploy-dev-for-pinniped.mk

```diff
@@ -50,6 +52,7 @@ delete-pinniped-jwt-authenticator:
 	kubectl --kubeconfig=${CLUSTER_CONFIG_FOR_PINNIPED} delete -f ./site/content/docs/latest/reference/manifests/kubeapps-pinniped-jwt-authenticator.yaml
 	kubectl --kubeconfig=${ADDITIONAL_CLUSTER_CONFIG_FOR_PINNIPED} delete -f ./site/content/docs/latest/reference/manifests/kubeapps-pinniped-jwt-authenticator.yaml
 
+# Using old version for deletion due to error for missing secret "pinniped-concierge-impersonation-proxy" not found
```

Comment:
Mmm, that's weird. Seems like an issue to report to our Pinniped friends, doesn't it?

Reply:
Not sure if that is a capability that they want to allow, but created issue [here](https://github.com/vmware-tanzu/pinniped/issues/1251).

Reply:
Thanks for reporting this! Please see https://github.com/vmware-tanzu/pinniped/issues/1251#issuecomment-1210935999.